### DISCUSSION TITLE
Duck audio of "main" window for OBC team radios

### DISCUSSION THREAD

**Discussion by BeyondVertical** [f1multiviewer/issue-tracker#200] [Feature requests]
It would be really awesome to have a ducking function for a selectable "main" window. For example the international feed.
When I have an OBC feed open and the driver or engineer is talking, then the audio of the international feed should be ducked or muted for that specific time, so that I can hear clearly what they are saying. This would be really cool. Also a visual indicator that team radio is active (I think this was a discussion already).

**Comment by BR4DY** [comment]
+1 for this, and being able to have multiple drivers windows open and duck the audio when one of them is speaking. I currently just hover the mouse over the mute button for the main window, so I can hear the on-board audio - but an auto-ducking feature would be amazing.
